pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,81 @@
+{ chafa
+, cmake
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, imagemagick_light
+, lib
+, libglvnd
+, libxcb
+, makeWrapper
+, ocl-icd
+, opencl-headers
+, pciutils
+, pkg-config
+, stdenv
+, vulkan-loader
+, wayland
+, xfce
+, xorg
```

Comment:
The packages provided by `xorg` should already be in scope, you can replace this with `libX11`, etc.